flink-dist/pom.xml

```diff
@@ -428,9 +411,6 @@ under the License.
 									<goal>single</goal>
 								</goals>
 								<configuration>
-									<descriptors>
-										<descriptor>src/main/assemblies/hadoop.xml</descriptor>
```

Comment:
you have to rework this to use the `maven-dependency-plugin` to download the jar beforehand, and then place it into the distribution, see https://github.com/apache/flink/pull/6167 for example on how this can be done.

Reply:
Yes, I think we should keep `include-hadoop` available. and we can download the jar by change the `hadoop.xml`. i.e.: `files` ->`dependencySet`.  and `Rework flink-dist assemblies` changes can in a new PR. What do you think?